contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -200,28 +206,54 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
 
```

Comment:
Maybe `require(hasInitialized() && !isPetrified(), "CONTRACT_NOT_INITIALIZED_OR_PETRIFIED");` at line 203 might be removed cause we have `_checkContractVersion(0);` check below?

Reply:
ok for me